tests/integration/scanner/cascade-nmap-sslyze.test.js

```diff
@@ -0,0 +1,36 @@
+const { cascadingScan } = require('../helpers')
+
+test(
+  "Cascading Scan nmap -> sslyze on unsafe-https",
+  async () => {
+    const { categories, severities, count } = await cascadingScan(
+      "nmap-unsafe-https-sslyze",
+      "nmap",
+      ["-Pn", "-sV", "unsafe-https", "-p 443"],
+      {
+        nameCascade: "https-tls-scan",
+        matchLabels: {
+          "securecodebox.io/invasive": "non-invasive",
+          "securecodebox.io/intensive": "light"
+        }
+      },
+      10 * 60
+    );
+
+    expect(count).toBe(4);
+    expect(categories).toMatchInlineSnapshot(`
+      Object {
+        "Invalid Certificate": 1,
+        "Outdated TLS Version": 2,
+        "TLS Service Info": 1,
+      }
+    `);
+    expect(severities).toMatchInlineSnapshot(`
+      Object {
+        "informational": 1,
+        "medium": 3,
+      }
+    `);
+  },
+  3 * 60 * 1000
```

Comment:
Ah sorry sebastian did'nt tell you about this,
we have basically two different types of timeouts, the first one set above in the cascadingScan() function call. And another one passed to jest, the test framework. 

The one for the test framework should always be higher than our timeout. This lets us fail with a better error message and print out the resources from the cluster in a more helpful way, to better debug the failing test.